### ISSUE TITLE
`not in` operator doesn't behave as expected

### ISSUE THREAD

**Issue by DasIch** [dag/attest#26]
I'm currently implementing a LazyList and for that I've this simple test to check if **contains** works:

```
l = LazyList(range(10))
5 in Assert(l)
'foo' not in Assert(l)
```

Now I expect this test to pass because 5 is in the list and well 'foo' is not but I get an AssertionError on the last line, if I do `'foo' in Assert(l)` that returns False but doesn't raise an AssertionError. Maybe I'm not seeing something here but this is totally weird in my opinion.

**Comment by dag** [comment]
http://packages.python.org/Attest/api.html#attest.Assert.in_

You can't override "not in", it just negates "in". If I'm wrong here and you know a way I'd love to hear it. For now you can use the not_in method.

**Comment by dag** [comment]
How about:

```
'foo' in AssertNot(l)
```

**Comment by dag** [comment]
In any case I should probably make this issue more clear in the docs.

**Comment by dag** [comment]
Warn in guide about some Assert operations, closed by afad047d62856c111b73818f986dbd39153def19
